app/Models/Game.php

```diff
@@ -128,19 +128,19 @@ public function gamefiles(){
             ->withTrashed();
     }
 
-    public function tags(){
+    public function tags() {
         return $this->hasMany('App\Models\TagRelation', 'content_id', 'id')->Where('content_type', '=', \DB::raw("'game'"))->with('tag');
     }
 
-    public function credits(){
+    public function credits() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->